composer.json

```diff
@@ -10,8 +10,11 @@
         }
     ],
     "minimum-stability": "stable",
+    "require": {
+        "php": "^7.0"
+    },
     "require-dev": {
-        "phpunit/phpunit": "^7.0.2"
+        "phpunit/phpunit": "^6.5 || ^7.0.2"
```

Comment:
The PHPUnit version `7.0+` only supports `php-7.1+`.

To be compatible with PHP version `7.0`, we need to add the `^6.5` additionally inside `require-dev` block.

Reply:
+1 :)